src/constants/translations/en/common.json

```
@@ -93,7 +93,8 @@
     "subject": "Please select a subject",
     "proficiencyLevel": "Please select at least one proficiency level",
     "youtubeLink": "Please provide a YouTube video link",
-    "currentAndNewPasswordsMatch": "Current and new passwords match."
+    "currentAndNewPasswordsMatch": "Current and new passwords match.",
```

We don't have dots in other error messages

```suggestion
    "currentAndNewPasswordsMatch": "Current and new passwords match",
```

---

This is not my fault, but I`ll change it